thriftbp/doc_client_test.go

```diff
@@ -87,11 +87,11 @@ func LoggingMiddleware(next thrift.TClient) thrift.TClient {
 	}
 }
 
-// This example demonstrates a typical use case of thriftclient pool in
+// This example demonstrates a typical use case of thriftbp pool in
 // microservice code with custom middleware.
-func Example() {
-	pool, err := thriftclient.NewBaseplateClientPool(
-		thriftclient.ClientPoolConfig{
+func Example_clientPool() {
```

Comment:
🔕 should this be `Example_ClientPool`?

Reply:
golint will yell at you if you try to capitalize the `_Suffix`

Reply:
Ok, I thought it might be something like that, thanks for confirming :)